test_requirements.txt

```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
```

Comment:
Without this `make test` fails with an opaque error if it is ever called before `make build` in the user's virtualenv. How about specifying `build` as a dependency of the `test` command in the make file, i.e.:

```
test: build
    flake8 --exclude ./lib/*
    python3 -m unittest tests/*.py
```